### PR TITLE
Update GOVERNANCE.md

### PR DESCRIPTION
**Pull request by phrobb**
This change fixes a few typos and suggests a couple of small content additions to the document.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Phil Robb Phil.Robb@est.tech -->

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ The responsibilities of the Technical Committee are
 * coordinating event participation and communications regarding the Eiffel Community Open Source Projects
 
 Eiffel Community aims to operate as a consensus based community, if any decision requires a vote to move the Eiffel Community Open Source Projects forward, the members of the Technical Committee will vote on a one vote per member basis.
-Simple majority is required for any decision to be approved.
+Simple majority is required for any decision to be approved. When voting in a meeting, it is a simple majority of the voting members present.  When voting through email or other forms of asynchronous communication, it is a simple majority of all TSC voting members.
```
Comment (e-backmark-ericsson) [inline GOVERNANCE.md:66]: What is 'TSC'? I assume you mean Technical Committee? I think we should avoid unnecessary abbreviations.

Comment (fdegir) [inline GOVERNANCE.md:66]:
```suggestion
Simple majority is required for any decision to be approved. When voting in a meeting, it is a simple majority of the voting members present.  When voting through email or other forms of asynchronous communication, it is a simple majority of all Technical Committee voting members.
```
TSC means Technical Steering Committee but we name the committee Technical Committee in Eiffel. We will need to document the role of the committee during bootstrap phase as part of the committee charter work to clarify it further. (will the committee do technical steering or just overseeing and so on.)

Also replaced TSC with Technical Committee.

File: GOVERNANCE.md [hunk 2]
```diff
@@ -122,10 +122,11 @@ The Bootstrap Technical Committee will be responsible for a series of specific a
 * Project guidelines
 * Code of conduct guidelines
 * Contribution process
+* Security/Vulnerability reporting and response processes
```
Comment (fdegir) [inline GOVERNANCE.md:125]: that's a great catch. I don't know how we missed this.

File: GOVERNANCE.md (hunk 1 above)
Comment (fdegir) [reply GOVERNANCE.md:66]: TSC means "Technical Steering Committee" but we name it "Technical Committee" in Eiffel. We will need to document the role of the committee during bootstrap phase as part of the committee charter work to clarify it further. (will the committee do technical steering or just overseeing and so on.)

Comment (e-backmark-ericsson) [reply GOVERNANCE.md:66]: 👍

**Review by fdegir — APPROVED** [review]
I think it would be good to merge this so we have Phil's updates available for the meeting on Monday.

**Review by henning-roos — APPROVED** [review]
Looks good to me!

**Comment by sodero** [comment]
LGTM

**Review by sodero — APPROVED** [review]

**Comment by raja-maragani** [comment]
Looks good to me.

**Review by raja-maragani — APPROVED** [review]

**Review by d-stahl-ericsson — APPROVED** [review]

**Review by k-hallen-ericsson — APPROVED** [review]

**Review by m-linner-ericsson — APPROVED** [review]
Looks good to me

**Pull request merged**